test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/ControllerSpec.scala

```diff
@@ -107,7 +145,8 @@ trait ControllerSpec extends WordSpec with Matchers with BeforeAndAfterAll with
     status(result)           shouldBe expectedStatus
 
     val doc = Jsoup.parse(contentAsString(result))
-    doc.select("h1").text should include(expectedTitle)
+    doc.select("h1").text   should include(expectedTitle)
+    doc.select("body").text should not include ("not_found_message(")
```

Comment:
it would be more helpful if the test logs said which message keys were missing. You could do something like this:
```scala
...
val regex = ???
val missingMessageKeys: List[String] = ??? // find all instances where regex is satisfied
if(missingMessageKeys.nonEmpty) fail("Message keys are missing: $missingMessageKeys") 
...
```

Reply:
Done.